rdkit/Chem/Draw/IPythonConsole.py

```diff
@@ -1,32 +1,22 @@
+#
+#  Copyright (C) 2011-2017 Greg Landrum
```

Comment:
It isn't a good idea to update copyrights on files that haven't been changed.  The copyright should reflect the first date of publication.  In the case of code changes, it is the code change that is being published.